src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/yflows/YFlowCreateSpec.groovy

```diff
@@ -191,11 +188,8 @@ class YFlowCreateSpec extends HealthCheckSpecification {
 
         and: "All involved switches pass switch validation"
         // https://github.com/telstra/open-kilda/issues/3411
-        northbound.synchronizeSwitch(yFlow.sharedEndpoint.switchId, true)
-        involvedSwitches.each { sw ->
-            northbound.validateSwitch(sw.dpId).verifyRuleSectionsAreEmpty(["missing", "excess", "misconfigured"])
-            northbound.validateSwitch(sw.dpId).verifyMeterSectionsAreEmpty(["missing", "excess", "misconfigured"])
-        }
+        switchHelper.synchronize(yFlow.sharedEndpoint.switchId)
```

Comment:
Because of https://github.com/telstra/open-kilda/issues/3411
If we synchronize all involved switches, we'll get discrepancy on shared endpoint (which is a known issue).